reactor-test/src/main/java/reactor/test/publisher/PublisherProbe.java

```diff
@@ -131,6 +131,12 @@ default void assertWasRequested() {
 	 * @return true if the probe was subscribed to at least once.
 	 */
 	boolean wasSubscribed();
+
+	/**
+	 * @return how many times probe was subscribed
+	 */
+	long subscribed();
```

Comment:
I would make this `long subscribeCount()`

Reply:
Ok